resources/db/migration/V1_20__add_event_application_review_state.sql

```diff
@@ -0,0 +1,9 @@
+
+-- Let's anticipate that we'll have many types of events for application, like "confirmation sent" etc. and prepare
+-- for that.
+-- Change old event_type to new_review_state (contains now 'received' only) and add event_type again which
+-- now tells what kind of event this is (later we will have others than 'review-state-change'
+alter table application_events rename column event_type to new_review_state;
```

Comment:
If we are going to have a generic `event_type` column then I think we should have a similar generic value for that. Like `event_value` or something more descriptive.

If we are going to stick with a non-generic/specific `new_review_state` column then at this point we could actually just drop the `event_type` column away since it doesn't have any kind of role. If the value of the `new_review_state` is set, then by that we already know that the event type is `review-state-change`. If the `new_review_state` is `NULL` then we know that the event type was something else.

If we instead would go with the generic solution and have just two columns, `event_type` and `event_value` (or then that more descriptive column name), then we don't have any need in the future to add new columns. We would then just be adding new values for the `event_type` column and - when needed - an associated value in the `event_value` column.